lib/std/zig/AstGen.zig

```diff
@@ -9120,7 +9113,18 @@ fn builtinCall(
             } else if (str.len == 0) {
                 return astgen.failTok(str_lit_token, "import path cannot be empty", .{});
             }
-            const result = try gz.addStrTok(.import, str.index, str_lit_token);
+            const res_ty = try ri.rl.resultType(gz, node) orelse .none;
```

Comment:
Imports now store result types.

Reply:
A possible ZIR optimization here would now be to replace the `rvalue` call below with special handling in the `ty` case, since Sema will do the coercion for us. Not a merge blocker, just worth noting!